components/tree-grid/docs.json

```diff
@@ -0,0 +1,7 @@
+{
+	"component": "tree-grid",
+	"status": "prod",
```

Comment:
This should be `prototype` until I can get a full accessibility review by a specialist.